docs/vm/create-windows-vm.md

```diff
@@ -101,3 +103,88 @@ Once all the settings are in place, click on `Create` to create the VM.
+    ![find-virtio-driver-directory](assets/find-virtio-driver-directory.png)
+    2. Click `OK` to allow the installer to scan this directory for drivers, choose `SUSE Block Driver for Windows`, and click `Next` to load the driver.
+    ![select-virtio-block-driver](assets/select-virtio-block-driver.png)
+    1. Wait for the installer to load up the driver. If you choose the correct driver version the `virtio` volumes will be detected once the driver is loaded.
+    ![installer-found-virtio-drive](assets/installer-found-virtio-drive.png)
+
+4. (Optional) If you are using other `virtio` based hardware like network adapter, you will need to install those drivers manually after completing the installation. To install drivers, open the VMDP driver disk, and use the installer based on your platform.
+
+The support matrix of VMDP driver pack for Windows are as follows (assume the VMDP CD-ROM drive path is E):
+
+| Version | Supported | Driver path |
+| :-----: | :-------: | :---------- |
+| Windows 7 | No | `N/A` |
+| Windows Server 2008 | No | `N/A` |
+| Windows Server 2008r2 | No | `N/A` |
+| Windows 8 x86(x64) | Yes | `E:\win8-2012\x86(x64)\pvvx` |
+| Windows Server 2012 x86(x64) | Yes | `E:\win8-2012\x86(x64)\pvvx` |
+| Windows 8.1 x86(x64) | Yes | `E:\win8.1-2012r2\x86(x64)\pvvx` |
+| Windows Server 2012r2 x86(x64) | Yes | `E:\win8.1-2012r2\x86(x64)\pvvx` |
+| Windows 10 x86(x64) | Yes | `E:\win10-server\x86(x64)\pvvx` |
+| Windows Server 2016 x86(x64) | Yes | `E:\win10-server\x86(x64)\pvvx` |
+| Windows Server 2019 x86(x64) | Yes | `E:\win10-server\x86(x64)\pvvx` |
+| Windows 11 x86(x64) | Yes | `E:\win10-2004\x86(x64)\pvvx` |
+| Windows Server 2022 x86(x64) | Yes | `E:\win10-2004\x86(x64)\pvvx` |
+
+!!! note
+    If you didn't use the `windows-iso-image-base-template` template, and you still need `virtio` devices, please make sure to add your custom Windows virtio driver to allow it to detect the hardware correctly.
+
+## Known Issues
+
+### Windows ISO unable to boot when using EFI mode
+
+When using EFI mode with Windows, you may find out the system booted with other devices like HDD or UEFI shell like the one below:
+
+![efi-shell](assets/efi-shell.png)
+
+That's because Windows will prompt a `Press any key to boot from CD or DVD...` to let the user decide whether to boot from the installer ISO or not, and it needs human interaction to allow boot from CD or DVD.
+
+![boot-from-cd](assets/boot-from-cd.png)
+
+As a workaround, if the system already booted into UEFI shell, you can type in `reset` to let the system reboot, once the prompt appears you can press any key to let system boot from Windows ISO.
+
+### VM crashes when reserved memory not enough
+
+There is a known issue with Windows VM, when allocated more than 8GiB of memory to Windows VM and not enough reserved memory configured, the VM crashes without warning. This can be fixed by allocating at least 256MiB of reserved memory to the on the `Advanced Options` tab. 
```

Comment:
There is a known issue with Windows VM when it is allocated more than 8GiB of without enough reserved memory configured. The VM crashes without warning.

Reply:
This can be fixed by allocating at least 256MiB of reserved memory to the template on the `Advanced Options` tab.